ui/public/locales/ar.json

```diff
@@ -93,7 +93,7 @@
 "label.action.delete.network": "Delete Network",
 "label.action.delete.network.processing": "Deleting Network....",
 "label.action.delete.nexusvswitch": "Delete Nexus 1000v",
-"label.action.delete.nic": "Remove NIC",
+"label.action.remove.nic": "Remove NIC",
```

Comment:
why is this change required? `delete` seems fine as a phrase.

Reply:
Everywhere else I saw this term as remove and not delete, and even the label text says remove. Futhermore, I think in ACS its more sensible to say remove instead of delete NIC, because in ACS the NICs cannot exist apart from a VM, it's not a separate entity, it can be added or removed from a VM, it cannot be created/destroyed in a vacuum. In the end, it's not that big of a deal, but it keeps the code cohesive.

Reply:
no, it isn´t a big deal, but a remove btween the deletes is as awkward as the delete with remove in the text. won´t :-1: , just want to say I don´t like this. it replaces noise with noise.
that said ; we should reorder these files alphabetically ;)